tools/pybind11Tools.cmake

```diff
@@ -51,10 +51,12 @@ endfunction()
 macro(_pybind11_return_if_cxx_and_linker_flags_work feature cxxflags linkerflags cxx_out linker_out)
   _pybind11_check_cxx_and_linker_flags(${cxxflags} ${linkerflags} check_success)
   if(check_success)
-    separate_arguments(cxxflags)
-    separate_arguments(linkerflags)
-    set(${cxx_out} ${cxxflags} PARENT_SCOPE)
-    set(${linker_out} ${linkerflags} PARENT_SCOPE)
+    set(cxx ${cxxflags})
+    set(linker ${linkerflags})
+    separate_arguments(cxx)
+    separate_arguments(linker)
```

Comment:
I meant it as written -- to avoid `set(linker ${linkerflags})`.

But actually, why is the transformation between space-separated and semicolon-separated lists even needed? Can't it be kept uniformly one or the other (`list(APPEND ...)`)?